packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -215,9 +242,9 @@ class DriveCommand extends RunCommandBase {
       if (error is ToolExit) {
         rethrow;
       }
-      throwToolExit('CAUGHT EXCEPTION: $error\n$stackTrace');
+      throw 'Unable to run test: $error\n$stackTrace';
```

Comment:
Make this throw an exception or error type instead of a String

Reply:
Done.